### DISCUSSION TITLE
Combined preview in webcontrol is very low-res 176*72px

### DISCUSSION THREAD

**Discussion by restorepoint1** [Motion-Project/motionplus#202] [General]
### Did you read the guide?

Yes

### What is the base version number of MotionPlus being used?

0.2.x

### What was the install method?

Built from source code

### What is base architecture?

ARM-32bit

### What is the distro being used?

Raspbian

### Disto version number

24.04

### Camera/Sound type(s) being used?

Network camera with RTSP

### Describe the issue/problem and steps to reproduce

"All Cameras" preview in webcontrol is very low-res 176*72px upscaled to fullscreen
![photo_2024-12-14_17-19-07](https://github.com/user-attachments/assets/af9f5e08-bdc9-4e0e-b763-745f69a5d632)

### Relevant MotionPlus log output (at log_level 8)

```shell
[NTC][ALL][ac00:allcam] init_validate: No stream_preview_params for cam 1
[NTC][ALL][ac00:allcam] init_validate: No stream_preview_params for cam 2
[NTC][ALL][ac00:allcam] init_validate: Creating default stream preview values
[DBG][ALL][ac00:allcam] init_cams: stream_preview_params values. Device 1 row 1 col 1
[DBG][ALL][ac00:allcam] init_cams: stream_preview_params values. Device 2 row 1 col 2
[DBG][STR][ac00:allcam] getsizes_scale: Device 1 Original Size 1920x1080 Scale 25 New Size 480x272
[DBG][STR][ac00:allcam] getsizes_scale: Device 2 Original Size 800x448 Scale 25 New Size 200x112
[DBG][STR][ac00:allcam] getsizes_pct: Combined Image Original Size 680x272 Scale 25 New Size 176x72
```

**Comment by Mr-Dave** [comment]
Adjust the `stream_preview_scale` within the `motion.conf` and all the `camera.conf` files to suit your needs.
